pkgs/development/python-modules/wsgiprox/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, six
+, certauth
+}:
+
+buildPythonPackage rec {
+  pname = "wsgiprox";
+  version = "1.5.2";
+
+  src = fetchFromGitHub {
+    owner = "webrecorder";
+    repo = "wsgiprox";
+    rev = "004870a87959e68ff28ff4362e4f0df28ec22030"; # Repo has no git tags
+    sha256 = "sha256-EquddaNrVceyJHuQMCajKHGZX2Q7ebR0Zhvi2pl2WEw=";
+  };
+
+  propagatedBuildInputs = [
+    six
+    certauth
+  ];
+
+  pythonImportsCheck = [ "wsgiprox" ];
+
+  # See https://github.com/webrecorder/wsgiprox/issues/6
+  doCheck = false;
```

Comment:
Can we run the tests which are passing?

Reply:
After replacing `--doctest-module` with `--doctest-modules` in setup.py, removing the https tests and using the checkPhase from 967e8cdde5a2e03d7de3edc236032179d4222781, I'm back to the point where some tests hang indefinitely. I don't find it easy enough to figure out which ones hang and I'd rather not spend any more time on this.